nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/admin/ClearCache.java

```diff
@@ -75,10 +75,15 @@
 @ExecuteOn(RuntimeType.INSTANCE)
 @TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CLUSTERED_INSTANCE, CommandTarget.CONFIG, CommandTarget.DEPLOYMENT_GROUP})
 @RestEndpoints({
-    @RestEndpoint(configBean = Domain.class,
-            opType = RestEndpoint.OpType.GET,
-            path = "lclear-cache",
-            description = "Clears a JCache or Hazalcast IMap")
+        @RestEndpoint(configBean = Domain.class,
+                opType = RestEndpoint.OpType.GET,
+                path = "clear-cache",
+                description = "Clears a JCache or Hazalcast IMap"),
+        @RestEndpoint(configBean = Domain.class,
+                opType = RestEndpoint.OpType.GET,
+                path = "lclear-cache",
+                description = "Clears a JCache or Hazalcast IMap")
+
```

Comment:
We can probably just drop it here in Community.
```suggestion
                description = "Clears a JCache or Hazalcast IMap")
```